Cargo.toml

```diff
@@ -175,7 +174,7 @@ members = ["crates/*"]
 anyhow = { version = "1", default-features = false }
 assert-json-diff = { version = "2", default-features = false }
 async-compression = { version = "0.3", default-features = false }
-async-nats = { version = "0.33", default-features = false }
+async-nats = { version = "0.35", git = "https://github.com/rvolosatovs/nats.rs", branch = "feat/command-sender", default-features = false }
```

Comment:
Refs https://github.com/nats-io/nats.rs/pull/1267

Looks like there's still no response on the PR, so I'll go ahead and publish `async-nats-wrpc` crate with this patch applied